dtests/src/test/java/io/snappydata/hydra/complexdatatypes/complexdatatypes.bt

```diff
@@ -0,0 +1,54 @@
+io/snappydata/hydra/complexdatatypes/arraytypes.conf
```

Comment:
Please change the conf file names as per hydra naming convention. e.g. arraytypes.conf should be arrayTypes.conf